### DISCUSSION TITLE
[@types/shelljs] at 0.8.15 has dependency errors

### DISCUSSION THREAD

**Discussion by mkelley33** [DefinitelyTyped/DefinitelyTyped#68277] [Issues with a @types package]
Node version (or tell us if you're using electron or some other framework):
v20.11.0

I'm using TypeScript if that makes a difference.

ShellJS version (the most recent version/Github branch you see the bug on):
"shelljs": "^0.8.5", and "@types/shelljs": "^0.8.15",

Operating system:
macOS Sonoma 14.1.1

Description of the bug:
node_modules/@types/shelljs/node_modules/@types/glob/index.d.ts:29:42 - error TS2694: Namespace '"myservice/node_modules/minimatch/dist/cjs/index"' has no exported member 'IOptions'.

29 interface IOptions extends minimatch.IOptions {
~~~~~~~~

node_modules/@types/shelljs/node_modules/@types/glob/index.d.ts:74:30 - error TS2724: '"myservice/node_modules/minimatch/dist/cjs/index"' has no exported member named 'IMinimatch'. Did you mean 'Minimatch'?

74 minimatch: minimatch.IMinimatch;
~~~~~~~~~~

Found 2 errors in the same file, starting at: node_modules/@types/shelljs/node_modules/@types/glob/index.d.ts:29

Example ShellJS command to reproduce the error:
import * as shell from 'shelljs';

shell.cp('-R', 'src/emails', 'build/src/');

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
This should solve the issue https://stackoverflow.com/a/77810047/2153237

**Reply by jpshack-at-palomar** [reply]
The slack overflow post cited above isn't really a solution for this problem. I have offered one here: https://stackoverflow.com/questions/75890950

**Comment by jpshack-at-palomar** [comment]
See https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/69867 for more detail on this. What I had thought was a thorough search somehow overlooked this issue.
